prometheus.yml

```diff
@@ -0,0 +1,29 @@
+# my global config
```

Comment:
Why is being being added as part of the `rc9` bump PR?

Reply:
Was unintended, forgot to add it to .gitignore